anypointmq/README.md

```diff
@@ -0,0 +1,290 @@
+# Anypoint MQ Bindings
+
+This document defines how to describe Anypoint MQ-specific information in AsyncAPI documents. 
+
+[Anypoint MQ](https://docs.mulesoft.com/mq/) is MuleSoft's multi-tenant, cloud messaging service and is fully integrated with [Anypoint Platform](https://www.mulesoft.com/platform/enterprise-integration).
+
+<a name="version"></a>
+## Versions
+
+The version of this bindings specification is `0.0.1`.
+This is also the `bindingVersion` for all binding objects defined by this specification.
+In any given binding object, `latest` MAY alternatively be used to refer to the currently latest published version of this bindings specification.
+
+The version of the AsyncAPI specification to which these bindings apply is `2.1.0`.
```

Comment:
Do we need to have this here? Why couldn't this binding be applied to a potential `2.2.0` or `2.1.1` version?

Reply:
> Do we need to have this here? Why couldn't this binding be applied to a potential `2.2.0` or `2.1.1` version?

i noticed that none of the other binding specs are tied to a spec version, but in writing the binding spec i definitely had in mind to close the gap between AsyncAPI `2.0.0` and the Anypoint MQ protocol. So i considered it prudent to state that. In truth the binding spec also applies to `2.1.0` as far as i can tell.

@fmvilas : If you are opposed to binding specs being explicitly tied to AsyncAPI spec versions, then i'll remove this. Let me know.

Reply:
I'm not opposed but I think we should align in the way we do it, so the rest of the bindings can do it in the same way. Would you mind opening a separate issue for that and removing this line for now?

Reply:
opened https://github.com/asyncapi/spec/issues/590

Reply:
and removed the line from the binding spec.